app/models/requested_topic.rb

```diff
@@ -0,0 +1,4 @@
+class RequestedTopic < ApplicationRecord
+  validates :name,
+            presence: true
```

Comment:
Gray and I have talked this over and come to the conclusion that it's OK to break past conventions if your pattern is better.